### ISSUE TITLE
Accessibility issue with ownerdrawn dialogs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [cuongnv764119/tortoisesvn#74]
```
The NVDA Screen reader (www.nvda-project.org) cannot read the message 
box-dialogs used in TortoiseSVN.
Examples:
- Success or error of creating a new local repository (context-menu ->
TortoiseSVN -> Create repository here)
- Clean up a working copy (context-menu -> TortoiseSVN -> Clean up)
- Revert an item (from within Check for modifications dialog)
- SVN Checkout into a directory that is not empty
Only the buttons and title of the dialog are recognized by NVDA, but not the 
actual message. Replacing these "ownerdrawn" dialogs with standard message 
boxes should fix the problem.
```

Original issue reported on code.google.com by `ralf1...@googlemail.com` on 17 Apr 2011 at 11:02
